corehq/apps/users/models.py

```diff
@@ -800,6 +813,8 @@ class CouchUser(Document, DjangoUserMixin, IsMemberOfMixin, UnicodeMixIn, EulaMi
     assigned_location_ids = StringListProperty()
     has_built_app = BooleanProperty(default=False)
 
+    reporting_metadata = SchemaProperty(ReportingMetadata)
```

Comment:
Right. You wouldn't assume

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/standard/monitoring.py#L296

I looked at the sort column and if it was username, i paginated the users then did the case query. Otherwise I grabbed the top x from the cases and then found the usernames